CoEpi/ui/healthquiz/QuestionCell.swift

```diff
@@ -14,6 +15,10 @@ class QuestionCell: UITableViewCell {
             this.onChecked?(q, checked)
         }
         accessoryView = checkBox
+
+        textLabel?.font = UIFontMetrics.default.scaledFont(for: Fonts.robotoRegular)
```

Comment:
This is still creating a new font per cell (unless the system caches it). Put it in `Fonts`.

Reply:
@i-schuetz It seems this only sets a reference to the existing font object.
![Screen Shot 2020-03-30 at 3 10 47 AM](https://user-images.githubusercontent.com/4785666/77867723-c1009f80-7238-11ea-9340-4addf9e3f443.jpg)

Reply:
Well, this is not cell specific. But I realized that caching the scaled font would prevent it from reacting to accessibility changes (which the user may do while they're using the app). So this should be fine.